src/code42cli/cmds/devices.py

```diff
@@ -103,7 +100,7 @@ def _change_device_name(sdk, guid, name):
 @sdk_options()
 def show(state, device_guid, format=None):
     """Print device info. Requires device GUID."""
-    _DEVICE_INFO_KEYS_MAP = OrderedDict()
```

Comment:
If we keep this in a method (like a helper method), the variable name should not be all-caps